sdktests/helpers_test.go

```diff
@@ -38,10 +39,10 @@ func TestComputeExpectedBucketValue(t *testing.T) {
 					p.userValue,
 					p.flagOrSegmentKey,
 					p.salt,
-					ldvalue.NewOptionalString(secondary),
+					o.Some(secondary),
 					p.seed,
 				)
-				failureDesc := selectString(secondary == "", "empty secondary key", "empty-but-not-undefined secondary key") +
+				failureDesc := h.IfElse(secondary == "", "empty secondary key", "empty-but-not-undefined secondary key") +
 					" should have changed result, but did not"
```

Comment:
For what it's worth, I do think the `Select` terminology was intuitive 🤷‍♂️

Reply:
My rationale was that the if { } else { } pattern implies correctly that the first part is what you get if it's true, which the word "select" does not.